client/src/components/Nav/NavProfile.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { NavItem } from 'react-materialize'
 import {isLogged} from '../../utils'
+import {Link} from 'react-router-dom'
```

Comment:
In accordance to style code [Add spaces inside curly braces](https://github.com/airbnb/javascript/#whitespace--in-braces)